Challenge2/build/index.html

```diff
@@ -0,0 +1,312 @@
+                                                    lg:w-300
+                                                    lg:ml-28
+                                                  "
+                                                >
+                                                  Contact Me
+                                                </button>
+                                              </div>
+                                        </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+        <!-- Fotter Section -->
+        <div class="h-20 bg-secondary">
+            <div class="flex flex-col ">
+                <div class="flex justify-between relative">
+                    <div class="bg-secondary 50 h-10 w-10"></div>
+                    <div class="h-10 w-56 flex justify-between">
+                        <div><a href="https://www.linkedin.com/in/ryam-almalki-a5b307201/"><svg class="h-8 w-6 text-white lg:h-8 lg:w-6 ml-1 "  viewBox="0 0 24 24"  fill="none"  stroke="currentColor"  stroke-width="2"  stroke-linecap="round"  stroke-linejoin="round">  <path d="M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2 2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6z" />  <rect x="2" y="9" width="4" height="12" />  <circle cx="4" cy="4" r="2" /></svg></a></div>
+                        <div><a href="https://github.com/RyamAlmalki"><svg class="h-8 w-6 text-white lg:h-8 lg:w-6  lg:ml-6"  viewBox="0 0 24 24"  fill="none"  stroke="currentColor"  stroke-width="2"  stroke-linecap="round"  stroke-linejoin="round">  <path d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22" /></svg></a></div>
+                        <div><a href=""><svg class="h-8 w-6 text-white lg:h-8 lg:w-6  lg:ml-6"  width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">  <path stroke="none" d="M0 0h24v24H0z"/>  <rect x="3" y="5" width="18" height="14" rx="2" />  <polyline points="3 7 12 13 21 7" /></svg></a></div>
+                    </div>
+                    <div class="h-8 w-8  hover:bg-white text-indigo-100 bg-pink-500 rounded-lg transition-colors duration-150 focus:shadow-outline">
+                        <a href="#Home"><svg class="h-8 w-8 hover:text-pink-500 text-white"  fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M5 15l7-7 7 7"/>
+                          </svg></a>
+                    </div>
+                </div>
+                <div class="flex justify-center relative">
```

Comment:
In the video i sent you where it explains positioning, using position fixed would help this button stay on screen so that you can always scroll back to top. Would help. Watch it.